integration_tests/jira_issues_tests.feature

```diff
@@ -61,7 +61,7 @@ Feature: Jira Issues Tests
   Scenario: D3ASIM-891, bids are reported correctly in csv export files
      Given we have a scenario named two_sided_pay_as_clear.default_2a
      And d3a is installed
-     When we run the d3a simulation on console with two_sided_pay_as_clear.default_2a
+     When we run the d3a simulation on console with two_sided_pay_as_clear.default_2a for 2 hrs
```

Comment:
How come you changed the duration here ? To decrease the integration tests' runtime ?

Reply:
If you have look at this [line](https://github.com/gridsingularity/d3a/pull/572/files#diff-0e919c3f7c79c6df9954c6557f70efa5L261), It was already running for 2 hrs. Just to use the same step for `self_sufficiency` test to be tested for one day, I made simulation runtime as a variable and set `2 hrs` for old tests.

Reply:
Thanks for clarifying this, looks great.